lib/modules/manager/bazel/artifacts.ts

```diff
@@ -171,16 +140,19 @@ export async function updateDependency({
         return null;
       }
 
-      let result = fileContent;
-      result = patchCodeAtFragments(result, urlFragments, updateValues);
-      result = updateCode(result, [idx, 'strip_prefix'], updateValues);
-      result = updateCode(result, [idx, 'sha256'], hash);
-      return result;
+      contents = patchCodeAtFragments(contents, urlFragments, updateValues);
+      contents = updateCode(contents, [idx, 'strip_prefix'], updateValues);
+      contents = updateCode(contents, [idx, 'sha256'], hash);
     }
-  } catch (err) /* istanbul ignore next */ {
```

Comment:
shouldn't we still catch errors?

Reply:
Good catch

Reply:
If I get it right, when we throw error from artifact update function, it means retry for the next run. When we return `artifactError` field, it means PR warning and retry can be forced manually by checking the checkbox.

The only thing that can throw here I believe is the HTTP request, but it's already catched. So I think this `catch` clause is redundant.

Reply:
Maybe let's just see which errors we can encounter here and adjust to more desired behavior eventually